src/test/java/com/nerodesk/om/aws/AwsAttributesTest.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2015, nerodesk.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the nerodesk.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.nerodesk.om.aws;
+
+import java.io.IOException;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Tests for {@link AwsAttributes}.
+ *
+ * @author Alexey Saenko (alexey.saenko@gmail.com)
+ * @version $Id$
+ * @since 0.4
+ */
+public final class AwsAttributesTest {
+
+    /**
+     * AwsAttributes can be invisible by default.
+     * @throws IOException If fails.
+     */
+    @Test
+    public void isNotVisibleByDefault() throws IOException {
+        final AwsAttributes attr = new AwsAttributes();
+        MatcherAssert.assertThat(attr.visible(), Matchers.equalTo(false));
+    }
+
+    /**
+     * AwsAttributes can change visibility status properly.
+     * @throws IOException If fails.
+     */
+    @Test
+    public void changesVisibility() throws IOException {
```

Comment:
@darkled Here too, let's declare `Exception` being thrown, as suggested [here](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names).